src/Host.cpp

```diff
@@ -1192,8 +1192,7 @@ void Host::readPackageConfig(const QString& luaConfig, QString& packageName)
         return;
     } else {
         // error
-        std::string e = "no error message available from Lua";
```

Comment:
I think this might an error - in other places with this code block the message assigned to `e` is only replaced by the error string on the top of the lua stack if there ***IS*** a message there - then the initial "there is not error message" makes perfect sense...

Reply:
`lua_tostring()` will _always_ return a value though, no?

Reply:
if there is nothing on the top of the stack then I'd expect it to return the equivalent to a lua `nil` - which at best might be an empty string - but I'm not the lua expert. Compare the code fragment here to the several instances at the end of `(void) TLuaInterpreter::initLuaGlobals()` ...

Reply:
Yeah, I am fairly certain it would be an empty string. It can't return anything else.

Reply:
So in that case we should return the message that there is no message - like we do elsewhere! :wink:

Reply:
Feedback applied.